therapy/schemas.py

```diff
@@ -338,14 +339,42 @@ class ValueObjectDescriptor(BaseModel):
     extensions: Optional[List[Extension]]
 
 
+class ServiceMeta(BaseModel):
+    """Metadata regarding the therapy-normalization service."""
+
+    name: str
```

Comment:
This shouldn't change, so we can just set it here. 
```suggestion
    name = 'thera-py'
```